### PR TITLE
DiferredJust for Single

### PR DESCRIPTION
**Pull request by Ipomoea**

### CODE REVIEW

File: Sources/Extensions/Single/Single+DeferredJust.swift [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+public extension Single {
+
+    /// Returns an single that invokes the specified factory function whenever a new observer subscribes.
+    ///
+    /// - Parameter elementFactory: Element factory function to invoke for each observer
+    /// that subscribes to the resulting sequence.
+    /// - Returns: An observable sequence whose observers trigger an invocation of the given element factory function.
+    static func deferredJust(_ elementFactory: @escaping () throws -> Element) -> Single<Element> {
+        return .create(subscribe: { (observer) -> Disposable in
```
Comment (petropavel13) [inline Sources/Extensions/Single/Single+DeferredJust.swift]: remove parentheses

Comment (petropavel13) [reply Sources/Extensions/Single/Single+DeferredJust.swift]: dot is not required

Comment (Ipomoea) [reply Sources/Extensions/Single/Single+DeferredJust.swift]: not compiles without dot

File: Sources/Extensions/Single/Single+DeferredJust.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+public extension Single {
+
+    /// Returns an single that invokes the specified factory function whenever a new observer subscribes.
+    ///
+    /// - Parameter elementFactory: Element factory function to invoke for each observer
+    /// that subscribes to the resulting sequence.
+    /// - Returns: An observable sequence whose observers trigger an invocation of the given element factory function.
+    static func deferredJust(_ elementFactory: @escaping () throws -> Element) -> Single<Element> {
+        return .create(subscribe: { observer -> Disposable in
```
Comment (scoreyou) [inline Sources/Extensions/Single/Single+DeferredJust.swift]: Could you use trailing closure syntax?
```swift
return .create { observer in
...
}

```

**Pull request merged**